File: python/ejercicios_todocode/writeline_n/writeline_n.py
#Programa que pida un numero maximo, y que luego muestre en terminal desde 1 hasta ese numero maximo

def main():
    n: int = int(input("Digita un valor: "))
    for i in range(1, n+1):
        print(i)
        
if __name__ == "__main__":
    main()

File: cpp/ejercicios_todocode/writeline_n/writeline_n.cpp
//Programa que pida un numero maximo, y que luego muestre en terminal desde 1 hasta ese numero maximo

#include <iostream>

int main(void)
{
    int n;
    std::cout << "Digita un numero: ";
    std::cin >> n;

    for (int i = 1; i <= n; i++)
    {
        std::cout << i << std::endl;
    }
    return 0;
}